site/content/docs/5.2/components/tags.md

```diff
@@ -25,6 +25,13 @@ Informative tags are built on top of `<span>` and are usually used to display ca
 
 For a list of tags of an article, for example, add a heading (`<h1>–<h6>`) to explain that we are in a tags list and use `<ul>` or `<ol>` depending on the use case.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
```

Comment:
OK. Then the design should add in the DSM the fact that the same way a tag can contain a icon, it can contain a number (when part of a numbered list). 
=> I'm removing all the callouts
=> I'm adding that point in my list for the design.

Reply:
Resolved ?